client/gutenberg/extensions/videopress/save.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import { createHigherOrderComponent } from '@wordpress/compose';
+import { RichText } from '@wordpress/editor';
+
+const VideoPressSave = CoreVideoSave => ( { attributes } ) => {
+	const { caption, guid } = attributes;
+
+	if ( ! guid ) {
+		return <CoreVideoSave attributes={ attributes } />;
```

Comment:
we may need to pass in more of the API here, or all of it…

```js
const VideoPressSave = CoreVideoSave => props => {
	const { attributes: { caption, guid } = {} } = props;

	if ( ! guid ) {
		return <CoreVideoSave { ...props } />
	}

	…
}
```

Reply:
Nice find! Unfortunately, this doesn't solve the the validation issue. Probably because the `save` component in core only uses the attributes: https://github.com/WordPress/gutenberg/blob/3f1324b53cc8bb45d08d12d5321d6f88510bed09/packages/block-library/src/video/index.js#L110

Even though, I applied your suggestion because it is what we should do in order to have the code working with any future change in the core `save` component that uses something else apart from the attributes.